.gitignore

```diff
@@ -5,7 +5,6 @@
 /build
 /captures
 .externalNativeBuild
-/gradle.properties
```

Comment:
Because I'm actually adding a `gradle.properties` files in this commit. In general, these should be under version control, in contrast to `local.properties` files.